src/views/Main/views/Join/components/Join.tsx

```diff
@@ -22,12 +22,12 @@ const Join = () => {
               {/* <li>
                 <a className="nav-link" href=""></a>
               </li> */}
-              <li>
+              /* <li>
```

Comment:
This doesn't work, you need to wrap this comment in `{...}`. See the above example of how to comment out JSX.

Reply:
OK